dbcon/execplan/simplecolumn.cpp

```diff
@@ -493,6 +493,167 @@ bool SimpleColumn::singleTable(CalpontSystemCatalog::TableAliasName& tan)
   return true;
 }
 
+void SimpleColumn::evaluateSimd(vector<uint32_t> &colList, vector<uint32_t> &colWidth, vector<vector<uint8_t>> &colData, uint32_t offset, uint32_t batchCount, SIMD_TYPE simdType) 
```

Comment:
I think this particular function is a bit too copy-pasty. Can we improve it by factoring out something?